### PR TITLE
Add PlatformSupport and GCCBootstrap artifacts for FreeBSD 12

### PR DESCRIPTION
**Pull request by ararslan**
Generated from https://github.com/JuliaPackaging/Yggdrasil/pull/3447.

### CODE REVIEW

**Comment by ararslan** [comment]
Actually, I think I need to delete the ones for FreeBSD 11, since CI seems to be picking those up. Not sure whether that's related to the failure in the privileged runner tests though.

**Comment by ararslan** [comment]
Ah, it can't find `PlatformSupport` because https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/master/src/Rootfs.jl#L512 is telling it to look specifically for `v"2021.1.28"` but for FreeBSD 12 we only have `v"2021.8.9"`. Nothing else has that though. Was I supposed to rebuild `PlatformSupport` for everything?

**Comment by giordano** [comment]
> Was I supposed to rebuild PlatformSupport for everything?

Oh, yes, it makes sense :smile:

File: Artifacts.toml [hunk 1]
```diff
@@ -2484,335 +2484,313 @@ os = "linux"
...
 lazy = true
 libc = "musl"
 os = "linux"
 
-    [["PlatformSupport-armv7l-linux-gnueabihf.v2021.1.28.x86_64-linux-musl.squashfs".download]]
-    sha256 = "eed3f5b7b5d64d8b0c7ebb3aaf27cc3b79fdbf21ca900655c8a2554bf6aabd49"
-    url = "https://github.com/JuliaPackaging/Yggdrasil/releases/download/PlatformSupport-v2021.1.28/PlatformSupport-armv7l-linux-gnueabihf.v2021.1.28.x86_64-linux-musl.squashfs.tar.gz"
+    [["PlatformSupport-armv7l-linux-gnueabihf.v2021.8.10.x86_64-linux-musl.squashfs".download]]
+    sha256 = "95d1e88aa9304b231971a9fee30353a1d08284410cc59be2e6a67ffabe385a1c"
+    url = "https://github.com/JuliaPackaging/Yggdrasil/releases/download/PlatformSupport-v2021.8.10+0/PlatformSupport-armv7l-linux-gnueabihf.v2021.8.10.x86_64-linux-musl.squashfs.tar.gz"
```
Comment (giordano) [inline Artifacts.toml:2562]: armv7l is using the `2021.8.10+0` version, the rest is `2021.8.10`

**Review by giordano — APPROVED** [review]
Looks good to me.  Tests are passing here and for [BinaryBuilder](https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/1032) (only failures  are known unrelated [Rust issues](https://github.com/JuliaPackaging/BinaryBuilder.jl/issues/1082)).  @staticfloat any comments?

**Review by staticfloat — APPROVED** [review]

**Pull request merged**